test/isUrlRequest.test.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const assert = require("assert");
+const loaderUtils = require("../");
+
+describe("isUrlRequest()", () => {
+	[
+		["test url", "about:blank", undefined, false],
+	].forEach(test => {
+		it("should isUrlRequest " + test[0] + " " + test[1] + " " + test[2], () => {
```

Comment:
The sentence you pass to `it()` should describe what you expect. You don't need to repeat `isUrlRequest` since you're already inside `describe("isUrlRequest()"`.

Remove the first item `"test url"` and change the `it()` call to

```js
it("should return " + test.pop() + " when called with " + test.join(), () => ...)
```